sycl-jit/passes/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # Module library for usage as library/pass-plugin with LLVM opt.
-add_llvm_library(SYCLKernelFusion MODULE
+add_llvm_library(SYCLKernelJit MODULE
```

Comment:
Should we maintain capitalization of "JIT" here, i.e., `SYCLKernelJIT`?

Reply:
Agreed, will apply the same logic to `SYCLKerneJitPasses`.